Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -474,7 +477,7 @@ def getContainerAttributes(String ds, Properties properties) {
 		def lastLevelQual = ds.tokenize('.').last()
 		if (properties.containerMapping) {
 			// obtain the deployType setting from the property
-			cMapping = evaluate(properties.containerMapping)
+			cMapping = parseJSONStringToMap(properties.containerMapping)
```

Comment:
`cMapping` is never properly defined as far as I can tell. Even if it works, I think this should read 'def cMapping = parse...'? Seems like that would be the best practice here

Reply:
@lburgess07 Good catch. Thanks for pointing that out. See new commit.